db/migrate/20120702003707_remove_settings.rb

```diff
@@ -14,9 +14,10 @@ def set(key, value)
 
 class RemoveSettings < ActiveRecord::Migration
   def up
-    results = ActiveRecord::Base.connection.select_rows('SELECT * FROM settings')
+    results = ActiveRecord::Base.connection.
+              select_rows("SELECT * FROM settings")
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.